lib/test/gopay_tokenization_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'veritrans'
+require 'minitest/autorun'
+
+class TestVeritrans < Minitest::Test
+  i_suck_and_my_tests_are_order_dependent!
+
+  def setup
+    @mt_test = Veritrans.new(
+      server_key: "SB-Mid-server-uQmMImQMeo0Ky3Svl90QTUj2",
+      client_key: "SB-Mid-client-ArNfhrh7st9bQKmz",
+      api_host: "https://api.sandbox.midtrans.com",
+      logger: Logger.new(STDOUT),
+      file_logger: Logger.new(STDOUT)
+    )
+  end
+
+  def test_link_payment_account
+    p 1
+    param = {
+      "payment_type": "gopay",
+      "gopay_partner": {
+        "phone_number": "81987654321",
+        "country_code": "62",
+        "redirect_url": "https://www.gojek.com"
+      }
+    }
+    result = @mt_test.link_payment_account(param)
+    assert_equal 201, result.status_code
+    assert_equal "PENDING", result.account_status
+    $test_gopay_id = result.account_id
+  end
+
+  def test_get_payment_account
+    p 2
+    param = {
+      "payment_type": "gopay",
+      "gopay_partner": {
+        "phone_number": "81987654321",
+        "country_code": "62",
+        "redirect_url": "https://www.gojek.com"
+      }
+    }
+    charge = @mt_test.link_payment_account(param)
+    result = @mt_test.get_payment_account(charge.account_id)
+    assert_equal 201, result.status_code
+    assert_equal "PENDING", result.account_status
+  end
+
+  def test_unlink_payment_account
+    p 3
+    result = @mt_test.unlink_payment_account($test_gopay_id)
+    assert_equal "Account status cannot be updated.", result.status_message
+  end
```

Comment:
is this expected that the test result are failing? if expected, rename it to e.g: `test_fail_to_unlink_...`